.drone.yml

```diff
@@ -1,38 +1,53 @@
 ---
-pipeline:
-  setup:
+kind: pipeline
```

Comment:
Rebase this ref. #6

Reply:
??. That PR was merged into this branch :smile:

Reply:
We don't need those commits twice. They are in master, so just rebase to get rid of them here.

Reply:
They aren't. But they should, so that #7 can have the correct config as well. I'll create a new PR and rebase this :ok_hand:

Reply:
Wut. Ops. Must have read something wrong 😅😅🙃